defaults/main/main.yml

```diff
@@ -41,6 +41,9 @@ pihole_dns_fqdn_required:     'false'
 pihole_dns_bogus_priv:        'false'
 pihole_dnssec:                'false'
 pihole_rev_server:            'false'
+pihole_rev_server_cidr:       "{{ ansible_default_ipv4.address | ansible.utils.ipaddr('network/prefix') }}"
+pihole_rev_server_target:     '{{ ansible_default_ipv4.gateway }}'
+pihole_rev_server_domain:     'lan'
```

Comment:
Based on documentation, this is optional; so I'd default this to an empty string.

https://docs.pi-hole.net/routers/fritzbox/#hostnames-instead-of-ip-addresses-in-pi-holes-web-interface-conditional-forwarding